examplebroker/broker.go

```diff
@@ -89,6 +89,7 @@ var (
 	exampleUsers   = map[string]userInfoBroker{
 		"user1":             {Password: "goodpass"},
 		"user2":             {Password: "goodpass"},
+		"user3":             {Password: "goodpass"},
```

Comment:
If the user does not require any special treatment, you can just use `user-integration-{something, something}` rather than adding another one to the ExampleBroker

Reply:
Ah, cool... I thought was needed as before was just getting error as the required password was ''